docs/hardware-provisioning.md

```diff
@@ -47,12 +47,6 @@ Storage tab of Spark's monitoring UI (`http://<driver-node>:4040`) to see its si
 Note that memory usage is greatly affected by storage level and serialization format -- see
 the [tuning guide](tuning.html) for tips on how to reduce it.
 
-Finally, note that the Java VM does not always behave well with more than 200 GB of RAM. If you
```

Comment:
You had a good point about SPARK_WORKER_INSTANCES being used in standalone mode. I think we might need to keep this doc as a result.